service/conf.ml

```diff
@@ -63,14 +63,58 @@ type platform = {
   opam_version : Ocaml_ci.Opam_version.t;
 }
 
+(* TODO Hardcoding the versions for now, this should expand to OV.Releases.recent.
+   Currently we only have base images for these 2 compiler variants. See ocurrent/macos-infra playbook.yml.
+*)
+let macos_distros : platform list =
+  [
+    {
+      label = "macos-homebrew";
+      builder = Builders.local;
+      pool = "macos-x86_64";
+      distro = "macos-homebrew";
+      ocaml_version = OV.Releases.v4_14;
+      arch = `X86_64;
+      opam_version = `V2_1;
+    };
+    {
+      label = "macos-homebrew";
+      builder = Builders.local;
+      pool = "macos-x86_64";
+      distro = "macos-homebrew";
+      ocaml_version = OV.Releases.v5_0;
+      arch = `X86_64;
+      opam_version = `V2_1;
+    };
+    (* Apple Silicon *)
```

Comment:
Even though we can support macOS arm64, we only have two machines available. @mtelvers

Reply:
Additional macOS arm64 machines will soon be added to the cluster.

Reply:
Additional macOS arm64 machines are available, I will enable both architectures.